### ISSUE TITLE
How To Get Sm64ex Coop Android v37

### ISSUE THREAD

**Issue by Klaskycs** [robertkirkman/sm64ex-coop#19]
**Describe the bug**
I Need Update My Sm64ex Coop Android To v37 Everyone Have It Except For Me Please Help Me
**Smartphone (please complete the following information):**
 - Device: [Honor x7]
 - OS: [Magic Os 7.1]

**Comment by robertkirkman** [comment]
Hello I have not created sm64ex-coop android v37 yet, but it might come here in the future. djoslin0's last update has a lot of things in it that can improve this repo, but it will take longer to sort through them than it took for his previous updates.

**Comment by Isaac0-dev** [comment]
fyi djoslin0 was not involved with the update sm64coopdx 1.0, he's put his statement on the public ex-coop repo

**Comment by robertkirkman** [comment]
> fyi djoslin0 was not involved with the update sm64coopdx 1.0, he's put his statement on the public ex-coop repo

my understanding is that actually, he wrote [this file](https://github.com/coop-deluxe/sm64coopdx/blob/main/src/pc/rom_assets.c). Do you have any corrections about that assumption?

**Comment by Klaskycs** [comment]
You Didn't Update It Everyone Is Mad

**Comment by zqpvr** [comment]
> You Didn't Update It Everyone Is Mad

it takes time...

**Comment by coop-tutorial** [comment]
> **Describe the bug** I Need Update My Sm64ex Coop Android To v37 Everyone Have It Except For Me Please Help Me **Smartphone (please complete the following information):**
> 
> * Device: [Honor x7]
> * OS: [Magic Os 7.1]

you can try this for as long as a quick alternative for your help

https://retired64.github.io/coopdx-android/

**Comment by robertkirkman** [comment]
Winlator is partially closed source software, but for many people who do not need their software to be open source, that is an option yes.

**Comment by Aust20156** [comment]
I Have v36.1 and I am Patiently Waiting.

**Comment by Aust20156** [comment]
> Winlator is partially closed source software, but for many people who do not need their software to be open source, that is an option yes.

my Winlator Doesn't Run on My FIRE HD 8 (10th gen)  please tell me somthing else

**Comment by Ryoikitenkaifukuma** [comment]
Bro roberkirkman you are the one who has the channel tacokitten in YouTube?

**Comment by robertkirkman** [comment]
> Bro roberkirkman you are the one who has the channel tacokitten in YouTube?

Yes that is where to see the tutorial for how to install sm64ex-coop version 36.1 on android

**Comment by frog8412** [comment]
you can try using mobox for this, i dont have a tutorial for this but i might create one later

**Comment by Ryoikitenkaifukuma** [comment]
Bro I want to play the game sm64excoop deluxe in primeos can I play if yes how I can?

**Comment by robertkirkman** [comment]
> Bro I want to play the game sm64excoop deluxe in primeos can I play if yes how I can?

Eventually I will send a .apk file for version 37 or later and it will have x86 support inside it so you will be able to play on PrimeOS but right now there is not version 37 available yet for the operating systems based on Surfaceflinger, DEX runtime, Bionic libc and Linux kernel  (Android distros), only version 36.1 which is called sm64ex-coop

**Comment by CarGamerYoutuber** [comment]
Okay What Happened

**Comment by infdevcarl** [comment]
hows the progress for dx?

**Comment by robertkirkman** [comment]
There are several places on github where I am posting experimental code tests in a series of iterative,  loosely interconnected repositories and code blocks that I will gradually finish and combine together into a future update for this repository. You can see some of them by looking at the repositories in my account and at some of the code I have posted in UNIX-like platform issues in the sm64coopdx repository. **None of the code is specifically connecting Mario 64 code to Android code yet**. It is system reorganization for both Android and Mario 64 that will help combine them together more seamlessly in the future once both of them are completely reorganized.
Stop posting comments specifically about Android on the sm64coopdx repository because they have stated that is not their focus and it's the responsibility of others to support Android, not them.

**Comment by robertkirkman** [comment]
**If you are a mobile C developer who edits the C or C++ code of this app on your own Android device, I could use some feedback for this question**:

Which of the following statements describes you most accurately?

A. I like running `make` to build my `libmain.so` and insert it into my `sm64.us.apk` using the traditional Termux build process, so that my edits to my `Makefile` are applied.

B. I like using Android Studio for Termux:X11 and prefer using the `gradle assembleDebug` and `ndk-build` commands, so that my edits to `Android.mk` files are applied.

[This page](https://github.com/lzhiyong/termux-ndk/issues/165) is also an acceptable place for further discussion on this topic. There is no plan to deprecate or remove either of those two options - the feedback I am looking for would tell me how to **improve and streamline both options**.

If you do not regularly edit the C or C++ code in this app then you do not need to mind this question.

**Comment by Elmigue04XD** [comment]
Hello, I have a problem, I downloaded sm64ex-coop from termux, I followed all the procedures and nothing happened, when I tried to open the application it crashed.

**Comment by GuythatUhmCreatesRandomStuff** [comment]
> > **Describe the bug** I Need Update My Sm64ex Coop Android To v37 Everyone Have It Except For Me Please Help Me **Smartphone (please complete the following information):**
> > 
> > * Device: [Honor x7]
> > * OS: [Magic Os 7.1]
> 
> you can try this for as long as a quick alternative for your help
> 
> https://retired64.github.io/coopdx-android/

i cant drag the rom file tho? I used many methods and it wont actually drag, even if i drag it from 7zip it wont let me run the game.

**Comment by DarknessBoy2010** [comment]
> > Bro I want to play the game sm64excoop deluxe in primeos can I play if yes how I can?
> 
> Eventually I will send a .apk file for version 37 or later and it will have x86 support inside it so you will be able to play on PrimeOS but right now there is not version 37 available yet for the operating systems based on Surfaceflinger, DEX runtime, Bionic libc and Linux kernel (Android distros), only version 36.1 which is called sm64ex-coop

https://github.com/ManIsCat2/sm64coopdx YOOO BRO GET YOURSELF REST THE ANDROID PORT IS RELEASED
